src/goto-symex/symex_other.cpp

```diff
@@ -62,8 +62,19 @@ void goto_symext::symex_decl(const expr2tc code)
   replace_dynamic_allocation(code2);
   replace_nondet(code2);
   dereference(code2, dereferencet::READ);
+  stack_limit = atol(options.get_option("stack-limit").c_str());
+  if(stack_limit > 0)
+  {
+    const std::string pretty_name =
```

Comment:
This looks dreadful (not your fault).

We need a separate function in another PR to clean up the names when writing these claims (memory leak claims and dead code claims also suffer from the same problem).

Reply:
I agree with @mikhailramalho 

Could we add the pretty name to our irep2 node?